news/rename_xthread.rst

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Rename: ``SpecModifier``, ``xthread``, ``xunthread`` renamed to ``DecoratorAlias``, ``@thread``, ``@unthread`` to support idea that
+  spec modifier is like a Python decorator.
```

Comment:
I don't see where `@thread` or `@unthread` is presented. I see some tests that construct such names, but nothing in the running code.

I fumbled trying to use `@thread` but was unsuccessful:

```
 @ $(@thread echo 'foo')
/var/folders/f2/2plv6q2n7l932m2x004jlw340000gn/T/pip-run-s1tb1kdf/xonsh/parsers/ast.py:143: DeprecationWarning: Constant.__init__ got an unexpected keyword argument 'is_raw'. Support for arbitrary keyword arguments is deprecated and will be removed in Python 3.15.
  return Constant(value=s, kind="str", **kwargs)
xonsh: subprocess mode: command not found: '@thread'
Did you mean the following?
    xthread:  Alias
''
```

Maybe I'm not using it right, but it sure looks to me like `xthread` is still the presented name. What am I missing?